examples/docker-rootful.yaml

```
@@ -44,16 +44,64 @@ provision:
     #!/bin/bash
     set -eux -o pipefail
     command -v docker >/dev/null 2>&1 && exit 0
-    if [ ! -e /etc/systemd/system/docker.socket.d/override.conf ]; then
-      mkdir -p /etc/systemd/system/docker.socket.d
+    readonly override_conf=/etc/systemd/system/docker.socket.d/override.conf
+    if [ ! -e "$override_conf" ]; then
+      mkdir -p $(dirname "$override_conf")
```

You didn't have to quote `$override_conf` because it is a fixed readonly constant that doesn't include whitespace. But if you do, then you should also quote the result of the `dirname` call:

```suggestion
      mkdir -p "$(dirname "$override_conf")"
```